include/psp2kern/kernel/threadmgr.h

```diff
@@ -94,14 +96,16 @@ typedef struct SceKernelThreadRunStatus {
 	} cpuInfo[4];
 } SceKernelThreadRunStatus;
 
-/* Sure there must be more than this, but haven't seen them */
 typedef enum SceThreadStatus {
-	SCE_THREAD_RUNNING = 1,
-	SCE_THREAD_READY   = 2,
-	SCE_THREAD_WAITING = 4,
-	SCE_THREAD_SUSPEND = 8,
-	SCE_THREAD_STOPPED = 16,
-	SCE_THREAD_KILLED  = 32, /* Thread manager has killed the thread (stack overflow) */
+	SCE_KERNEL_THREAD_STATUS_RUNNING   = 1,
```

Comment:
Could you leave previous name for compatiblity ?

SCE_THREAD_...

Reply:
Yes, done.